### ISSUE TITLE
[CLI] Local docs command

### ISSUE THREAD

**Issue by virkt25** [strongloop/loopback-next#1233]
## Description / Steps to reproduce / Feature proposal

As we now have a `@loopback/docs` package, I think it might be a good idea to leverage it so they can get a local copy of the docs up and running (helps in offline development). 

I'm thinking of a command like `lb4 docs` which can stand up a local copy of the site using Jekyll. It can take an option version parameter matching the version of the `@loopback/docs` package version.

Possibly related to https://github.com/strongloop/loopback-next/issues/1232

**Comment by shimks** [comment]
In my mind, running a local copy of the docs should ideally be accomplished by our script and not our CLI tool. I'd reason that baking that feature into the CLI has no real purpose to the tooling's intended userbase.

**Comment by shimks** [comment]
Interesting, I thought the feature would be for the developers of `loopback-next` to easily test out whether their newly written docs would be properly rendered or not.

Originally, I was referring to adding in a script in the monorepo to achieve the function that I was thinking about (testing proper docs rendering), but I now can see the appeal in making this feature available for the users as well through our CLI.

TLDR; +1

**Comment by dhmlau** [comment]
probably fixed in https://github.com/strongloop/loopback-next/pull/1254 ?

**Comment by virkt25** [comment]
#1254 sets up a possible foundation for this task but it is not the same as this task. This task is about enabling the `lb4` CLI to have a `docs` command to spin up a local copy of the docs site whereas #1254 is more about testing the `@loopback/docs` package to make sure nothing breaks.

**Comment by jannyHou** [comment]
> I'm thinking of a command like lb4 docs which can stand up a local copy of the site using Jekyll.

Not very expert of Jekyll(implementation details), but preview the site on local sounds great to me.

**Comment by bajtos** [comment]
Removing from GA, please let me know if anybody disagrees.
